### ISSUE TITLE
prohardver.hu anti-adblock

### ISSUE THREAD

**Issue by AmigoBarba** [uBlockOrigin/uAssets#6714]
### URL(s) where the issue occurs

```
https://prohardver.hu/index.html
https://mobilarena.hu/index.html
https://itcafe.hu/index.html
https://gamepod.hu/index.html
https://logout.hu/index.html
https://hardverapro.hu/index.html
```

### Describe the issue

Anti-adblock pops up on these websites. It can be skipped, but after a while it pops up again.
The websites are part of one site, each site is about a different area (IT hardware, mobile phones, IT news etc).

### Screenshot(s)

https://i.imgur.com/BShOXJs.png

### Versions

- Browser/version: Firefox 71.0 (64bit)
- uBlock Origin version: 1.24.2

### Settings

- [List here all the changes you made to uBO's default settings]

### Notes

No need to sign in. I have NanoDefender properly set up, but it doesn't work on these sites.

**Comment by mapx-** [comment]
working filter:
`gamepod.hu,itcafe.hu,prohardver.hu##+js(nostif, , 7000)`

I get no anti adb on the last 2 sites.
`mobilarena` is already fixed in uBlock filters annoyance (you have to enable it)

**Comment by mapx-** [comment]
For other issues with hungarian sites you could try to report here:
https://github.com/hufilter/hufilter/issues

**Comment by AmigoBarba** [comment]
Thanks for the filter. Will this be added by default in the future?

I would have contacted the hufilter team with this issue, but the above site owner's asked to whitelist them.
Thanks again.

**Comment by mapx-** [comment]
As you can see above `closed this in d40db72 17 hours ago` the filter was already added but in `uBlock filters - annoyance` list, just enable it

![image](https://user-images.githubusercontent.com/10303732/71320882-27e02100-24b2-11ea-9bea-a4c1deadde8c.png)

**Comment by mapx-** [comment]
However, is strange, `hufilter team` already added to the list filters to circumvent anti adb stuff, maybe they are saying the sites are not really blocked (only annoyances) and wont add filters for such minor stuff
